src/Microsoft.OData.Edm/Validation/ValidationRuleSet.cs

```diff
@@ -34,7 +34,7 @@ public sealed class ValidationRuleSet : IEnumerable<ValidationRule>
                 ValidationRules.EntityTypeInvalidKeyNullablePart,
                 ValidationRules.EntityTypeEntityKeyMustBeScalar,
                 ValidationRules.EntityTypeInvalidKeyKeyDefinedInBaseClass,
-                ValidationRules.EntityTypeKeyMissingOnEntityType,
+////                ValidationRules.EntityTypeKeyMissingOnEntityType,

```

Comment:
>ValidationRules.EntityTypeKeyMissingOnEntityType, [](start = 20, length = 49)

remove?

Reply:
I wanted to leave it here, explicitly commented out, so we knew we intended it to be disabled.

---
In reply to: [174975195](https://github.com/OData/odata.net/pull/1109#discussion_r174975195) [](ancestors = 174975195)